### ISSUE TITLE
Spurious logging-too-many-args (PLE1205) if using the msg kwarg

### ISSUE THREAD

**Issue by adamtheturtle** [astral-sh/ruff#6454]
`logging.info`, `logging.warning`, `logging.error`, etc. each take `(msg, *args, **kwargs)`.
If `msg` is given as a keyword argument, ruff 0.0.283 gives an error (https://beta.ruff.rs/docs/rules/logging-too-many-args/), which is incorrect.

```python
# example.py

import logging

logging.info(msg="X")
```

```
ruff --select=PLE --isolated example.py
```

```
example.py:3:1: PLE1205 Too many arguments for `logging` format string
```

**Comment by charliermarsh** [comment]
Thanks, will get this fixed shortly.
